beamerthemeuulm.sty

```diff
@@ -262,6 +262,15 @@
 % ---------------------------------------------
 
 
+% ---------------------------------------------
+% handling lectures
+% ---------------------------------------------
+\AtBeginLecture{
+    \part{\insertlecture}\label{lec:\thelecture}
+}
```

Comment:
I am missing subtitles at the beginning of each lecture (could indicate the topic of each lecture)